reverso_context_api/client.py

```diff
@@ -179,6 +217,21 @@ def _process_fav_entry(self, entry, cleanup):
             processed_entry[field_to] = val
         return processed_entry
 
+    def _process_history_entry(self, entry, cleanup):
+        entry_fields_map = {
+            "srcLang": "source_lang",
+            "srcText": "source_text",
+            "trgLang": "target_lang",
+            "translation1": "translation_1",
```

Comment:
All translation variants are returned in fields "translationN". There can be many of them, I think they have to be extracted and put into list field "translations". Also sometimes there are even empty values, which have to be removed.